# Description: edquota_editor does not get installed into /usr/sbin in Debian.
# Author: Michael Meskes <meskes@debian.org>

--- quota-3.16/ldap-scripts/applySystemQuotas.pl.orig	2008-06-24 16:53:12.000000000 +0200
+++ quota-3.16/ldap-scripts/applySystemQuotas.pl	2008-06-24 16:53:58.000000000 +0200
@@ -24,7 +24,7 @@
 my $Port = '389';
 my $BindDN = 'cn=Manager,dc=borgia,dc=com';
 my $SSL = 0;
-my $edquota_editor = '/usr/sbin/edquota_editor';
+my $edquota_editor = '/usr/share/quota/ldap/edquota_editor';
 my $edquota = '/usr/sbin/edquota';
 
 my $b = '';
